backend/src/nodes/impl/pytorch/model_loading.py

```diff
@@ -1,15 +1,16 @@
 from sanic.log import logger
 
+from .architecture.face.codeformer import CodeFormer
+from .architecture.face.gfpganv1_clean_arch import GFPGANv1Clean
+from .architecture.face.restoreformer_arch import RestoreFormer
+from .architecture.HAT import HAT
+from .architecture.LaMa import LaMa
 from .architecture.RRDB import RRDBNet as ESRGAN
```

Comment:
Man, we need something that enforces sorted imports. What are you using? It seems to be quite good.

Reply:
It's probably isort, right?

Reply:
I think it's isort? It's whatever VSCode uses when I do right click > sort imports, and at some point the isort extension got installed. I think maybe the python extension started installing isort alongside itself?